CMakeLists.txt

```diff
@@ -1,20 +1,21 @@
 cmake_minimum_required(VERSION 3.13)
-cmake_policy(SET CMP0003 NEW)
+#cmake_policy(SET CMP0003 NEW)
 project(picoquic C CXX)
 find_package (Threads REQUIRED)
 
 option(ENABLE_ASAN "Enable AddressSanitizer (ASAN) for debugging" OFF)
 option(ENABLE_UBSAN "Enable UndefinedBehaviorSanitizer (UBSan) for debugging" OFF)
 
-set(CMAKE_C_STANDARD 11)
+#set(CMAKE_C_STANDARD 11)
 
 message(STATUS "Initial CMAKE_C_FLAGS=${CMAKE_C_FLAGS}")
 
-if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    set(CMAKE_C_FLAGS "-std=c99 -Wall -O3 -fno-exceptions -fno-signed-zeros -fno-trapping-math ${CC_WARNING_FLAGS} ${CMAKE_C_FLAGS}")
-else()
-    set(CMAKE_C_FLAGS "-std=c99 -Wall -O3 -fno-exceptions -fno-signed-zeros -frename-registers -fno-trapping-math ${CC_WARNING_FLAGS} ${CMAKE_C_FLAGS}")
-endif()
+# Set compile options below
+#if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+#    set(CMAKE_C_FLAGS "-std=c99 -Wall -O3 -fno-exceptions -fno-signed-zeros -fno-trapping-math ${CC_WARNING_FLAGS} ${CMAKE_C_FLAGS}")
+#else()
+#    set(CMAKE_C_FLAGS "-std=c99 -Wall -O3 -fno-exceptions -fno-signed-zeros -frename-registers -fno-trapping-math ${CC_WARNING_FLAGS} ${CMAKE_C_FLAGS}")
+#endif()
 
```

Comment:
Would be nice to test for "Clang OR AppleClang". The issues come from `-frename-registers`, which I think is GCC only.